src/benchmarking.rs

```diff
@@ -3,18 +3,18 @@
 use super::*;
 
 #[allow(unused)]
-use crate::Pallet as Template;
+use crate::Pallet as LivingAssetsOwnership;
```

Comment:
Are these changes related to the Clippy linting at all?

Reply:
yes, the command

```bash
cargo clippy  --all-targets --workspace --features runtime-benchmarks --features try-runtime
```
also checks for code that's under `runtime-benchmarks` flag.